packages/plugin/src/theia.d.ts

```diff
@@ -12207,6 +12207,610 @@ export module '@theia/plugin' {
     }
 }
 
+/**
+ * Namespace for testing functionality. Tests are published by registering
+ * {@link TestController} instances, then adding {@link TestItem TestItems}.
+ * Controllers may also describe how to run tests by creating one or more
+ * {@link TestRunProfile} instances.
+ */
+export namespace tests {
+    /**
+     * Creates a new test controller.
+     *
+     * @param id Identifier for the controller, must be globally unique.
+     * @param label A human-readable label for the controller.
+     * @returns An instance of the {@link TestController}.
+     * @stubbed
+     */
+    export function createTestController(id: string, label: string): TestController;
+}
+
+/**
+ * The kind of executions that {@link TestRunProfile TestRunProfiles} control.
+ */
+export enum TestRunProfileKind {
+    /**
+     * @stubbed
+     */
+    Run = 1,
+    /**
+     * @stubbed
+     */
+    Debug = 2,
+    /**
+     * @stubbed
+     */
+    Coverage = 3,
+}
```

Comment:
Should we mark this as stubbed? I see you've put the implementation in the `types-impl` file rather than the `stubs` area, and the values are correct, so the enum itself is completely implemented, and a future implementer of the full API wouldn't need to do any more work here. Same question re: `TestTag`, `TestRunRequest`, and `TestMessage`.

Reply:
> Should we mark this as stubbed? I see you've put the implementation in the `types-impl` file rather than the `stubs` area, and the values are correct, so the enum itself is completely implemented, and a future implementer of the full API wouldn't need to do any more work here. Same question re: `TestTag`, `TestRunRequest`, and `TestMessage`.

@colin-grant-work, I tried the way you propose, it seems clear enough in the report since the main triggers remain marked as `@stubbed`, 
So I posted a new commit reflecting your proposal.

Thanks!!